src/build_workflow/build_target.py

```diff
@@ -55,7 +55,7 @@ def opensearch_version(self) -> str:
         )
 
     @property
-    def compatible_opensearch_versions(self) -> List[str]:
+    def compatible_core_versions(self) -> List[str]:
```

Comment:
Let's rename this back to `compatible_opensearch_versions`, we don't use "core" anywhere.

Reply:
Will it be confusing if I utilize this within the OpenSearch Dashboards plugin check?

Reply:
Oh I see! I think the right name for this would be `min` then?